src/main.ts

```
@@ -157,24 +158,31 @@ async function handleFailedRecording(): Promise<void> {
+        console.error(
+            'Meeting failed:',
+            error instanceof Error ? error.message : error,
+        )
 
-        const errorMessage =
-            error instanceof JoinError
-                ? error.message
-                : 'Recording failed to complete'
+        // Use global error if available, otherwise fallback to error message
+        const errorMessage = GLOBAL.hasError()
+            ? GLOBAL.getError()!.message
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)